docker/compose/public_full_node/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+# This compose file defines a Public Fullnode docker compose wrapper around libra-node. Currently
+# this can only connect to a local swarm instance, but work is in progress to enable more use
+# cases.
+#
+# In order to use, place a copy of the proper genesis.blob and waypoint.txt in this directory.
+#
+# Copy libra/config/src/config/test_data/public_full_node.yaml to this directory.
+# Modify it such that:
+# genesis_file_location: "/opt/libra/etc/geensis.blob"
+# rpc address: "0.0.0.0:8080"
+# waypoint from_config to from_disk: "/opt/libra/etc/waypoint.txt"
+#
+# TODO:
+#   * Directions on the correct image
+#   * Place to obtain waypoints and genesis.blob
+#   * Any modifications / specifying seed_addrs
+#
+# Additional information:
+# * If you use this compose for different Libra Networks, you will need remove the db volume first.
+version: "3.8"
+services:
+    fullnode:
+        image: libra/validator:test-2_8be5f4fa
```

Comment:
Probably should add more comments around here, if someone downloads an up to date master, they should be able to just pull the default or master. If they modify it, they'll need to use the docker file. Will save that for later.

Reply:
Hmm test-2 is just a throw away release I built when changing/testing circleci configs/scripts.   I wouldn't use it here as it will be deleted in 2 weeks.    Perhaps use a release?

Reply:
A wrapping script could discover the version of the network, provided there is a discovery mechanism.

Reply:
That's fair. I am oppose to adding scripts here. Scripts only run on Linux and Mac. It doesn't eliminate the need for documentation and only adds more since we'll have to document the scripts usage as well. If we're going to introduce scripts, we should eliminate docker compose and consider docker altogether.

With that being said, if we end up with something so CI can test it, sure, but that will remain undocumented and not a supported tool.